torch/_dynamo/output_graph.py

```diff
@@ -454,6 +465,10 @@ def register_attr_or_module(
         assert not isinstance(source, ParamBufferSource)
 
         if isinstance(target, torch.Tensor):
+            if len(self.tracers) > 1:
+                unimplemented(
+                    "accessing attribute of nn.Module inside HigherOrderOperator"
+                )
```

Comment:
1) Why here?
2) You may have to let constant source through, or add a comment about constant source.

Reply:
> Why here?

I'm not sure where else to do it. I've considered doing it in SubgraphTracer.create_proxy when it sees a `get_attr` call, but we can't actually tell what the get_attr call is for (e.g. if it is for a Tensor, or for a body function (which is represented as a GraphModule)) without adding more infra.

> You may have to let constant source through, or add a comment about constant source.

Good point, let me play around with it.

Reply:
@voznesenskym do you have an example of what it means to have constant source? I am not sure how to construct a repro that gets here